### DISCUSSION TITLE
Tables: nested field in body keys

### DISCUSSION THREAD

**Discussion by balassal** [skeletonlabs/skeleton#2202] [Q&A] [answered]
I would like to ask, how can i achive the following. I have e.g.
```
const tableSource: TableSource = {
    head: ['PID', 'State', 'Last Modified'],
    body: tableMapperValues(data.projects, ['pid', 'state', 'last_modified']),
    foot: ['Total Projects', '', `<code class="code">${data.projects.length}</code>`]
 }
```
What if the "state" key is an object: {"state": "live", "info": ""} and i would like to use state.state as key? How can i do this?
Thank you!

**Comment by Sarenor** [comment, marked as answer]
You can't. You either have to map it yourself or use a custom ``<table>`` that you build by hand instead of the ``<Table>`` component.

**Reply by endigo9740** [reply]
Here's the docs for setting up a custom table that uses the Skeleton theme: https://www.skeleton.dev/elements/tables

**Reply by balassal** [reply]
Thank you for your answer!
